.pre-commit-config.yaml

```diff
@@ -15,3 +15,8 @@ repos:
       rev: v17.0.6
       hooks:
           - id: clang-format
+    - repo: https://github.com/muya/dart-format-pre-commit # See https://github.com/Cretezy/dart-format-pre-commit/pull/1
+      rev: b1c57eb1447a2d25c618017576b94793fd7aa37c
```

Comment:
Bear in mind that if muya ever force-pushes that branch to e.g. update the PR, or if the PR is merged in any way that isn't adding a merge commit, that rev will go away.

Reply:
yes, maintainer seem slow as hell, but was "official" repo for this hook listed on pre-commit web page :(

did not find any better alternatives. everything is shit as always with these things